recipes/naif_eop_historical/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "naif_eop_historical" %}
+{% set version = "2024.8.28.1" %}
+{% set python_min = "3.9" %}
```

Comment:
We only need to set `python_min` if the min version is higher than conda-forge's min. 
```suggestion
```